src/services/api.ts

```diff
@@ -165,8 +165,8 @@ export class BaseApi {
   ): Promise<string> {
     let request = window.require("request");
     let options = {
-      timeout: 5000,
-      pool: { maxSockets: 10 },
+      timeout: 10000,
+      pool: { maxSockets: 1 },
```

Comment:
Hmm, only 1? Is that enough?

Reply:
I'm doing some research and testing with this; 1 seems to be working fine, anyways 5 should be fine too.